debian/control

```diff
@@ -1,7 +1,7 @@
 Source: plight
 Maintainer: Greg Swift <greg.swift@rackspace.com>
 Section: misc
-Build-Depends: python-all (>= 2.6.6-3), python-setuptools, debhelper (>= 9)
+Build-Depends: python (>= 2.6.6-3), python-setuptools, debhelper (>= 9)
```

Comment:
it kept failing with unmet dependencies.  I figured that python-all is not the name of the package. After changing from python-all to python, the package would build.